plugins/services/src/js/components/forms/MultiContainerFormAdvancedSection.js

```diff
@@ -36,7 +36,7 @@ const MultiContainerFormAdvancedSection = ({ data, path }) => {
         Advanced Settings
       </Trans>
       <Trans render="p">Advanced settings of the container.</Trans>
-      <p>{getForcePullSection(data, path)}</p>
+      <div className="form-group">{getForcePullSection(data, path)}</div>
```

Comment:
This is improved, but I don't think a div tag is allowed inside a p, either... only "phrasing content" is allowed according to the spec, which is defined as

![Screen Shot 2019-05-09 at 3 58 19 PM](https://user-images.githubusercontent.com/174332/57489477-60105780-7273-11e9-8aea-9413303953a2.png)

Reply:
This makes the parents a div and the p is inside it.